### ISSUE TITLE
Possible ios autoplay workaround

### ISSUE THREAD

**Issue by CSomerville** [JukeboxApp/voxpop#43]
Hey I just saw this. Could be a matter of starting the web audio context and piping the stream api to the destination.

https://github.com/tambien/StartAudioContext

**Comment by klee0589** [comment]
Im trying to wrap my head around this web audio context. Could you branch off of dev and test it out?

**Comment by fyodorjung** [comment]
Hey, Kang Il. I just saw CSomerville's comment. Any development with this or has it not been tried / confirmed yet. (I know you said you've moved on to just having a "mothership," but if this could be a gamechanger then I might be interested in giving it a shot.)

**Comment by klee0589** [comment]
Yes, this is definetly a game changer. We are so far unsuccessful bypassing the user gesture requirement on mobile but if you would like to give it a shot, Im all for it. Let me know if you need anything from my end. What did you have in mind?
